docs/language-agnostic-definition.md

```diff
@@ -1,7 +1,13 @@
-An Autonomous Economic Agent is, in technical terms, defined by the following characteristics:
+Currently, there is an implementation of the AEA framework in Python which enables the development of AEAs in Python, and allows AEAs which are built with it to run.
+
+However, AEAs, by design, are not constrained by any technical requirement, including the choice of programming language. This is further backed by the idea that agent-based solutions are suited for multi-stakeholder environments where the different AEAs may be developed independently of one another, resulting in a heterogeneous system. 
```

Comment:
```suggestion
However, AEAs can be developed in many different programming languages. This is further backed by the idea that agent-based solutions are suited for multi-stakeholder environments where the different AEAs may be developed independently of one another, resulting in a heterogeneous system. 
```

Reply:
There are technical constraints of course.

Reply:
Done